rust-bindings/sys/src/lib.rs

```diff
@@ -309,62 +309,6 @@ impl ::std::fmt::Debug for OstreeAsyncProgressClass {
     }
 }
 
-#[repr(C)]
-pub struct _OstreeBootloader {
-    _data: [u8; 0],
-    _marker: core::marker::PhantomData<(*mut u8, core::marker::PhantomPinned)>,
-}
-
-pub type OstreeBootloader = *mut _OstreeBootloader;
```

Comment:
BTW @tintou this looks good - we were leaking these pointless private structs into the API before.